package/network/utils/umbim/files/lib/netifd/proto/mbim.sh

```diff
@@ -155,7 +155,7 @@ _proto_mbim_setup() {
 
 	local connect_state
 	echo "mbim[$$]" "Connect to network"
-	connect_state=$(umbim $DBG -n -t $tid -d $device connect "$req_pdptype:$apn" "$auth" "$username" "$password") || {
+	connect_state=$(umbim $DBG -n -t $tid -d $device connect "$req_pdptype$apn" "$auth" "$username" "$password") || {
```

Comment:
How is this change related to the commit message? 
For me it looks like you should have done this in the previous commit already.

Reply:
You're correct, that's a mistake when reordering commits. I'll push the fix when I figure out the MTU and MTU6 thing.

Reply:
I decided not to wait with pushing this fix, while I try to figure out the MTU thing. I don't see any helpers for this in netifd repostiory either, so it's going to take some time.